BLSQ/iaso
# IA-2094: User columns in Forms submissions

User columns in Forms submissions

Related JIRA tickets : IA-2094

## Self proofreading checklist


## Changes

 added new default column in instances

## How to test

Open instances page and check that you have created by option in possible columns, activate it, sort by this column, ...

## Print screen / video

<img width="1652" alt="Screenshot 2023-04-24 at 14 54 34" src="https://user-images.githubusercontent.com/12494624/234005039-cd85c93f-448d-49b6-9c24-28168430bb7f.png">
<img width="437" alt="Screenshot 2023-04-24 at 14 54 26" src="https://user-images.githubusercontent.com/12494624/234005047-fb55eb69-3bb1-46c0-b5c5-b93d8dc7a9ad.png">
